test.cpp

```diff
@@ -0,0 +1,33 @@
+#include <assert.h>
+#include <vector>
+#include <tuple>
+#include "detect_range.hpp"
+
+int main()
+{
+    std::vector<int> current = {
```

Comment:
Test cases need have variations in test data for eg. {3} {} {3,4,5,5,7,9,10} - include repeated numbers, single entry list etc.